order/README.md

```diff
@@ -44,3 +45,58 @@ for GET /orders
 | ---------- | ------------------------- | ---------------------------------------- |
 | orders     | \[Order Object\]          | **REQUIRED.** A list of orders.          |
 | links      | Map\<object, Link Object> | **REQUIRED.** Links for e.g. pagination. |
+
```

Comment:
If the `GET /orders/{orderId}/status` endpoint is implemented, there must be a link to the endpoint using the relation type `monitor`.